crates/suiop-cli/src/cli/utils.rs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Mysten Labs, Inc.
```

Comment:
Let's actually move this to the [`cli/lib/` module](https://github.com/MystenLabs/sui/tree/9a83c7b9cca67247acbf91ce806d68a34dc23dc8/crates/suiop-cli/src/cli/lib) since that has been our catchall utils dir.